xdrip-get-entries.sh

```diff
@@ -822,12 +807,6 @@ function initialize_mode()
   #  to reflect in LSR in order to make it safer to allow seamless transition to LSR calibration
   mode="native-calibrates-lsr"
 
-  if [[ "$cmd_line_mode" == "expired" ]]; then
-    mode="expired"
-  fi
```

Comment:
I don’t think we’re ready to remove support for expired mode yet.

Reply:
Agreed - I'll add this back in to the modes branch and we can handle that one in a separate branch.